eva/catalog/catalog_manager.py

```diff
@@ -116,6 +117,73 @@ def create_metadata(
         metadata.schema = column_list
         return metadata
 
+    def create_video_metadata(self, name: str) -> DataFrameMetadata:
+        """Create video metadata object.
+            We have predefined columns for such a object
+            id:  the frame id
+            data: the frame data
+
+        Arguments:
+            name (str): name of the metadata to be added to the catalog
+
+        Returns:
+            DataFrameMetadata:  corresponding metadata for the input table info
+        """
+        columns = [
+            ColumnDefinition(
+                "name", ColumnType.TEXT, None, [], ColConstraintInfo(unique=True)
+            ),
+            ColumnDefinition("id", ColumnType.INTEGER, None, []),
```

Comment:
Do we still want to use a explicit `id` column or change to `_row_id` column?

Reply:
@jiashenC The function is moved out from the binder for clarification (since it is not used in binder anymore). No modifications are made. Regarding the id vs _row_id, I do not have preference, either works.

Reply:
Sure, just get your opinion if we can get rid of this redundant column. Others LGTM.

Reply:
Just come to my mind, under the current implementation. id and _row_id are different when a dataset has more than 1 video. id restates from 0/1 for ever video. _row_id probably should not.